internal/clients/team3/client.go

```diff
@@ -36,6 +36,9 @@ type client struct {
 	speakerPerformance   map[shared.ClientID]int
 	presidentPerformance map[shared.ClientID]int
 
+	// ## IIGO ##
+	ruleVotedOn string
```

Comment:
Is this value set anywhere? I can't seem to find it. And what is it used for?

Reply:
It is use to know what is the rule voted on for this turn. We need it to retrieve what we voted for this turn. The method where it would be set was not implemented at the time we implemented this, I should be able to update it now. Do you think this variable should be declared somewhere else?

Reply:
Yes, this could be stored in `c.iigoInfo.ruleVotingResults[{rule name}].ourVote`. I believe we can vote for multiple votes in the same turn (@Neelesh99 @preetl ?) so this could be a way of doing it.

Reply:
There is only one vote on rules per turn :) I think it was something we wanted to do but didn't do in the end.